README.md

```diff
@@ -16,7 +19,7 @@ Crawler to fetch data from SNIRH
 _WINDOWS_
 
 ```bash
-git clone https://github.com/franciscobmacedo/snirhcrawler
+git clone https://github.com/uematsusoft/snirhcrawler
```

Comment:
Could you avoid tracking this changes in the README?

Reply:
Hi, 
Stupid question: how can I remove the readme commits from the PR? Shouls I cancel the PR and then make another with only the first two commits?

Reply:
You can do it in multiple ways just remove the commits that edit the README using interactive rebase:

https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#:~:text=Changing%20Multiple%20Commit%20Messages

You can then force push and the PR would be updated.
However, this will actually remove the changes in the README from your machine, that maybe you don't want to. If you want to keep those changes in your local machine, you can just reset the 4 commits with soft:

```bash
git reset --soft HEAD~4
```

this will reset the commit history of the last 4 commits but not discard any changes. You can now just create again the 2 relevant commits, and not add the changes in the README.